### PR TITLE
tftpy refactor

### PR DESCRIPTION
**Pull request by sarnold**
* add tftp server module and daemon script
* add repolite cfg and toxfile for tox extension

### CODE REVIEW

File: pyserv/tftpd.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    return tftpd_server
+
+
+def tftpd_run(iface='', port=8069, directory=DOCROOT):  # pragma: no cover
+    """
+    Run in foreground command wrapper for TFTP console entry point;
+    init server listening, run the server, stop the server.
+
+    :param app: WSGI app (defaults to ``demo_app`` if not provided)
+    :param port: WSGI server listen port
```
Comment (SJLC) [inline pyserv/tftpd.py:37]: param comments don't match

File: .repolite.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+prog_name: repolite
+top_dir: ext  # local directory path for enabled repositories
+pull_with_rebase: false  # use --ff-only if false
+# add new repo_name sections as needed
+repos:
+  - repo_name: py3tftp
```
Comment (SJLC) [inline .repolite.yml:6]: Think this is not used, remove?

File: pyserv/tftpd.py (hunk 1 above)
Comment (sarnold) [reply pyserv/tftpd.py]: Thanks, fixed.

File: .repolite.yml (hunk 2 above)
Comment (sarnold) [reply .repolite.yml:6]: Actually I put both packages there for test/comparison as needed.  As of now tftpy looks okay using the last release from pypi so no need for syncing repositories just yet.

**Pull request merged**